packages/sit-onyx/.storybook/pages/introduction.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta } from "@storybook/blocks";
+import logoDark from "../../../../.github/onyx-logo-dark.svg";
+import logoLight from "../../../../.github/onyx-logo-light.svg";
+import logo from "../../../storybook-utils/src/assets/logo-onyx.svg";
+
+{/* workaround for hiding the table of contents, see docs-template.scss file */}
+
+<div className="storybook-hide-toc"></div>
+
+<Meta title="Introduction" />
+
+<img alt="onyx logo" src={logo} width="256px" style={{ display: "block", marginBottom: "2rem" }} />
+
+# Introduction
+
+Welcome! This is the Storybook for the onyx design system where you can see and play around with all our components.
+
+Are you looking for the general information about onyx? Then please head over to our [onyx documentation](https://onyx.schwarz).
```

Comment:
```suggestion
Are you looking for general information about onyx? Then please head over to our [onyx documentation](https://onyx.schwarz).
```

Reply:
hm, if someone uses storybook.onyx.schwarz, it might be better to make the onyx.schwarz link an `<a>` with target _blank